mne/viz/ica.py

```diff
@@ -787,6 +787,8 @@ def _plot_sources_raw(ica, raw, picks, exclude, start, stop, show, title,
     inds = list(range(len(picks)))
     data = np.array(data)
     n_channels = min([20, len(picks)])
+    raw._first_time = raw.first_samp / float(raw.info['sfreq'])
```

Comment:
@Eric89GXL perhaps I misunderstood something in your comment [here](https://github.com/mne-tools/mne-python/pull/3729#discussion_r86401457). Isn't this what you were suggesting?